### PR TITLE
PDCT 998/ensure new documents have relationships in old and new schema

### PR DESCRIPTION
**Pull request by diversemix**
# Description

- Upgrade dbclient to v3.2.1 & remove ruff config
- Configure pyright to work with pyenv in makefile
- Add pyrightconfig to gitignore & add trunk ignores
- Add ruff config for trunk
- Populate old and new schema for new families.
- Don't add corpus type on family create.
- Update blank.sql with alembic version 0032 schema
- Fix configure pyright & pyenv script
- Use new function scoped db fixture that runs the db migrations ⚠️ - this is 3x slower than before :(

Co-authored-by: @diversemix 

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Patch
- [x] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: .trunk/configure-pyright-with-pyenv.sh [hunk 1]
```diff
@@ -7,22 +7,19 @@ venv_name=$(grep -m 1 venv pyproject.toml | tr -s ' ' | tr -d '"' | tr -d "'" |
...
 		echo "pyenv not installed. Please install pyenv..."
 		exit 1
 	fi
 
-	# Check if pyenv-pyright plugin is installed
-	if ! command -v pyenv pyright &>/dev/null; then
-		echo "pyenv-pyright not installed. Installing..."
-		pyenv_root=$(pyenv root)
+	pyenv_root=$(pyenv root)
+	if [ -z $(ls -A "${pyenv_root}"/plugins/pyenv-pyright) ]; then
```
Comment (diversemix) [inline .trunk/configure-pyright-with-pyenv.sh:16]: If you type `man test` you'll find:
```
       -d FILE
              FILE exists and is a directory
```
which may be a little neater

File: integration_tests/family/test_create.py [hunk 2]
```diff
@@ -24,28 +25,47 @@ def test_create_family(client: TestClient, data_db: Session, user_header_token):
...
-    import_id = response.json()
-    assert import_id == "CCLW.family.i00000001.n0000"
-    actual_family = data_db.query(Family).filter(Family.import_id == import_id).one()
+    expected_import_id = "CCLW.family.i00000001.n0000"
+    assert response.json() == expected_import_id
+    actual_family = (
+        data_db.query(Family).filter(Family.import_id == expected_import_id).one()
+    )
 
     assert actual_family.title == "Title"
```
Comment (diversemix) [inline integration_tests/family/test_create.py:34]: Do we also want to test the `actual_family.organisation_id` here too - just to ensure its still being used'?

File: .trunk/configure-pyright-with-pyenv.sh (hunk 1 above)
Comment (katybaulch) [reply .trunk/configure-pyright-with-pyenv.sh:16]: I specifically want to test that the dir exists **and** that the dir isn't empty. Not sure I can do that with just `test`?

File: integration_tests/family/test_create.py (hunk 2 above)
Comment (katybaulch) [reply integration_tests/family/test_create.py:34]: Good shout

**Comment by diversemix** [comment]
All looking good from my side of the :person_fencing:

**Review by katybaulch — APPROVED** [review]
Peter raised, so I need to close it off 😆 Lovely work from us both if I do say so 😉

**Pull request merged**